pkg/symbol/symbol.go

```diff
@@ -62,16 +62,16 @@ func (s *Symbolizer) Symbolize(prof *profiler.Profile) error {
 	}
 
 	pid := prof.PID
-	userFunctions, err := s.resolveJITedFunctions(pid, prof.UserLocations)
+	userJITedFunctions, err := s.resolveJITedFunctions(pid, prof.UserLocations)
 	if err != nil {
 		// We expect only a minority of processes to have a JIT and produce the perf map.
 		if !errors.Is(err, perf.ErrNotFound) {
 			level.Debug(s.logger).Log("msg", "failed to obtain perf map for pid", "pid", pid, "err", err)
 			return nil
 		}
-		return fmt.Errorf("failed to resolve user functions: %w", err)
+		return fmt.Errorf("failed to resolve user JITed functions: %w", err)
```

Comment:
Added this to clarify that this is not a 'normal' user address but it's a JIT'ed one